### PR TITLE
ADA1-4

### PR DESCRIPTION
**Pull request by ch55secake**
Please review

### CODE REVIEW

File: AgentDiscoveries-Frontend/app/sass/styles.scss [hunk 1]
```diff
@@ -76,4 +76,12 @@ body {
 
 .profile-img-container:hover .overlay {
   opacity: 0.8;
+}
+
+.aTable {
```
[the remark this answers is not in the document]
Comment (ch55secake) [reply AgentDiscoveries-Frontend/app/sass/styles.scss:81]: No its does nothing, something I tried whilst trying to fix a bug feel free to delete it within resolution

File: AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/LocationsRoutes.java [hunk 2]
```diff
@@ -65,6 +66,8 @@ public Location updateEntity(Request req, Response res, int id) {
         if (location.getLocationId() != id && location.getLocationId() != 0) {
             throw new FailedRequestException(ErrorCode.INVALID_INPUT, "locationId cannot be specified differently to URI");
         }
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/LocationsRoutes.java:70]: Same deal here - extra line

File: AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx [hunk 3]
```diff
@@ -12,19 +12,21 @@ export default class Entity extends React.Component {
 
     render() {
         return (
-            <tr key={this.id}>
+            <tr style={{minWidth: '400px'}} key={this.id}>
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx:15]: We might not need this anymore now we have the fix below

File: AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx [hunk 4]
```diff
@@ -12,19 +12,21 @@ export default class Entity extends React.Component {
...
-
+    
     getEntityRow() {
-        return Object.keys(this.props.entity).map(key =>
-            <td key={key}>{this.props.entity[key].toString()}</td>);
+        const possibleColumns = ['locationId','siteName','location','timeZone', 'regionId','latitude','longitude'];
+        return possibleColumns.map(columnName =>
+            <td key={columnName}>{this.props.entity[columnName] ? this.props.entity[columnName].toString() : '-'}</td>);
     }
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx:29]: Extra new line here

File: AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx [hunk 5]
```diff
@@ -12,6 +12,9 @@ export default class LocationForm extends React.Component {
             location: '',
             timeZone: '',
             regionId: '',
+            latitude: '',
+            longitude: '',
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx:18]: Probably get rid of these spare lines

**Review by RogVan — CHANGES_REQUESTED** [review]

**Pull request merged**